crates/matrix-sdk-ui/src/room_list_service/room_list.rs

```diff
@@ -29,12 +32,15 @@ use super::{Error, State};
 #[derive(Debug)]
 pub struct RoomList {
     sliding_sync_list: SlidingSyncList,
+    entries_batch_drainer: Sender<()>,
```

Comment:
How about `Observable<()>` instead?

Reply:
Damn…!! Of course yes! We don't need to spawn a new task, we already have a `Subscriber<State>`, which is exactly what we need. It simplifies things so much. Thanks!